uaclient/entitlements/tests/conftest.py

```diff
@@ -3,51 +3,68 @@
 from uaclient import config
 
 try:
-    from typing import Any, Dict, List  # noqa
+    from typing import Any, Dict, List, Optional  # noqa
 except ImportError:
     # typing isn't available on trusty, so ignore its absence
     pass
 
 
-def machine_token(entitlement_type: str) -> "Dict[str, Any]":
+def machine_token(
+    type: str,
+    *,
+    affordances: "Dict[str, Any]" = None,
+    directives: "Dict[str, Any]" = None,
+    entitled: "Optional[bool]" = True,
+    obligations: "Dict[str, Any]" = None,
+    suites: "List[str]" = None
+) -> "Dict[str, Any]":
     return {
+        "resourceTokens": [{"Type": type, "Token": "%s-token" % type}],
```

Comment:
A nit, but `.format` here would be more consistent with the rest of the codebase.